src/fake_useragent/utils.py

```diff
@@ -49,43 +43,10 @@ def load() -> list[BrowserUserAgentData]:
         list[BrowserUserAgentData]: The list of browser user agent data, following the
             `BrowserUserAgentData` schema.
     """
-    data = []
-    ret: Union[list[BrowserUserAgentData], None] = None
     try:
-        json_lines = (
-            ilr.files("fake_useragent.data").joinpath("browsers.jsonl").read_text()
+        return json.loads(
+            f"[{get_data('fake_useragent', 'data/browsers.jsonl').rstrip().decode().replace(chr(10), ',')}]"
```

Comment:
I would write `"\n"` instead of the more cryptic `chr(10)`.

Reply:
That's because python 3.9 don't accept `\` inside f-strings' `{}`

Reply:
I'd separate it into a an assignment then to avoid the convoluted oneliner which ends up being harder to read. Something like

```python
raw_data = get_data(__package__, "data/browsers.jsonl").rstrip().decode()
comma_joined_data = raw_data.replace("\n", ",")
json_list_string = f"[{comma_joined}]"
return json.loads(json_list_string)
```
might do the trick. Otherwise, please add a comment!